### DISCUSSION TITLE
İBAX TEAM

### DISCUSSION THREAD

**Discussion by ByTeachers** [IBAX-io/go-ibax#1977] [Q&A] [answered]
I want the IBAX team information to be added to the site. Thus, we have the opportunity to get to know them better. Thanks.

**Comment by yolandadadada** [comment, marked as answer]
IBAX official website already has information about the team.

**Reply by ByTeachers** [reply]
Thank you for your instant responses to questions.

**Reply by ByTeachers** [reply]
I love this team. I hope we've drawn attention to the team. :)
